components/text_length_filter/src/main.py

```diff
@@ -0,0 +1,39 @@
+"""A component that filters out text passages which are to short."""
+import logging
+
+import pandas as pd
+
+from fondant.component import PandasTransformComponent
+from fondant.logger import configure_logging
+
+configure_logging()
+logger = logging.getLogger(__name__)
+
+
+class TextLengthFilterComponent(PandasTransformComponent):
+    """A component that filters out text passages which are to short."""
+
+    def setup(self, *, min_length: int):
+        self.min_length = min_length
+
+    def transform(self, dataframe: pd.DataFrame) -> pd.DataFrame:
+        """
+        Filter out text passages which are to short.
+
+        Args:
+            dataframe: Pandas dataframe.
+
+        Returns:
+            Pandas dataframe.
+        """
+        mask = (
+                dataframe["text"]["data"].apply(lambda x: len(x.split()))
```

Comment:
This filters on the amount of words, right? This isn't really clear from the documentation of the components or arguments. I would expect `min_length` to define the number of characters based on the current description.

Reply:
Yes as an FYI the DataComp competition filters text based on 2 things: minimum amount of characters and minimum amount of words: https://github.com/mlfoundations/datacomp/blob/3ac587eefa16e57e4494f18024894fa642cd1a9f/baselines/apply_filter.py#L67-L75

Maybe we can include the same functionality here.